datasync/chaindatafetcher/kafka/config.go

```diff
@@ -41,6 +41,7 @@ const (
 	DefaultMaxMessageBytes      = 1000000
 	DefaultRequiredAcks         = 1
 	DefaultSegmentSizeBytes     = 1000000 // 1 MB
+	DefaultConsumerBufferSize   = 100
```

Comment:
```suggestion
	DefaultConsumerBufferSize   = 100  // mas number of messages in buffer
```
If `DefaultConsumerBufferSize` can be set by a user, it would be nice to put recommanded number. Maybe the recommanded number for `DefaultConsumerBufferSize` is (number of partitions) * (average number of segments) * 100 ?

Reply:
Reflected. c62d77c